radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -219,6 +219,40 @@ elseif(PCB STREQUAL X7)
     add_definitions(-DRADIO_T8)
     add_definitions(-DEEPROM_VARIANT=0x4004)
     add_definitions(-DMANUFACTURER_RADIOMASTER)
+
+  elseif(PCBREV STREQUAL Commando8)          #Modified T8 applies to commando8
+    option(INTERNAL_MODULE_MULTI "Support for MULTI internal module" ON)
+    set(EXTERNAL_MODULE YES)
+    set(AFHDS3 NO)
+    set(FLAVOUR Commando8)
+    set(NAVIGATION_TYPE x7)
+
+    # set(CPU_TYPE STM32F2)
+    # set(CPU_TYPE_FULL STM32F205xE)  # for size report
+    # set(LINKER_SCRIPT targets/taranis/stm32f2_flash.ld)
+
+    set(CPU_TYPE STM32F4)
+    set(CPU_TYPE_FULL STM32F407xE)  # for size report
+    set(LINKER_SCRIPT targets/taranis/stm32f4_flash.ld)
+    set(FIRMWARE_TARGET_SRC
+      ${FIRMWARE_TARGET_SRC}
+      startup_stm32f40_41xxx.s
+      )
+    add_definitions(-DSTM32F40_41xxx)
+    set(ROTARY_ENCODER NO)
+    set(LUA_EXPORT lua_export_Commando8)
+    set(BLUETOOTH NO)
+    set(UNRTCLOCK YES)
+    if(iFlight_RELEASE)
+      set(BIND_KEY NO)
+    endif()
+    add_definitions(-DRADIO_Commando8)
+    add_definitions(-DCommando8_Float_Show_OFF) #Commando8_Float_Show_OFF
+    add_definitions(-DCommando8_STICKS_PWM_EN) #Commando8_STICKS_PWM_EN
+    add_definitions(-DiFlight_RELEASE)
+    add_definitions(-DEEPROM_VARIANT=0x4008)
+    add_definitions(-DMANUFACTURER_iFlight)#add_definitions(-DMANUFACTURER_RADIOMASTER)
+    add_definitions(-DCHANNEL_1023)
```

Comment:
@3djc can you please have a look at the defines above? That does not really look like the way it is done normally.

Reply:
yeah, need to work on a proper version, this is fairly broken too (check for iFlight_RELEASE and set it later)